lib/forms/Form.js

```diff
@@ -177,29 +177,81 @@ function MLForm$$createForm(schema, hostObject, formData, template) {
     if (schema.css)
         form.css.config = schema.css;
 
-    // allow schema to define confined CSS per form
-    if (schema.style) {
-        // an inspector might change the form element document
-        // if available, wait for its 'formshown' signal
-        var
-            inspector = hostObject && hostObject.inspector,
-            id = form.el.id || ('ml-form-' + counter++),
-            onFormShown = function () {
-                form.style = restyle('#' + id, schema.style, [], form.el.ownerDocument);
-                form.el.id = id;
-            };
+    // an inspector might change the form element document
+    // if available, wait for its 'formshown' signal
+    const inspector = hostObject && hostObject.inspector,
+        id = form.el.id || ('ml-form-' + counter++),
+        onFormShown = function () {
+            // allow schema to define confined CSS per form
+            form.style = restyle('#' + id, Object.assign({
```

Comment:
So my only thought is that here, we will be making it impossible to override these tooltip styles. I don't mind the styles being brought in specifically for each form, even if it is a bit wasteful. But with this unique id, it will be very difficult to override the styles without changing milo-ui itself. Or maybe I've misunderstood.

Reply:
Or is it possible to provide an ID so you know what you need to override. Maybe these generic styles shouldn't use #id.

Reply:
You can override it with the `schema.style` option? So that way you can tweak it on a form by form basis, as it uses Object.assign all you need to do is to use the same keys to override them. This means to override it you don't have to go anywhere near milo-ui, just the form schema that you're using.